app/RequirementTests/tests.hs

```diff
@@ -9,18 +9,32 @@ module Main
 (  main  ) where
 
 import Control.Monad
+import Config (databasePath)
```

Comment:
You can now take this opportunity to do some file reorganization.

1. Create a new folder called `app/Tests`, and move this file (`tests.hs`) to a new file `RequirementsTests.hs` (a module), and export those tests.
2. Add your new test file `CourseControllerTests.hs` under the new folder.
3. Create a new file `Main.hs` under the new folder, have it import the tests from both `RequirementsTests.hs` and `CourseControllerTests.hs`, and define the `main` function there.
4. Make sure to change the cabal file so that the new `Tests/Main.hs` file is the main entrypoint for running the tests.